pkg/reconciler/pipelinerun/resources/pipelineref_test.go

```diff
@@ -568,61 +565,68 @@ func TestGetPipelineFunc_VerifyNoError(t *testing.T) {
 	requesterUnsignedMatched := test.NewRequester(resolvedUnsignedMatched, nil)
 
 	testcases := []struct {
-		name                      string
-		requester                 *test.Requester
-		verificationNoMatchPolicy string
-		pipelinerun               v1beta1.PipelineRun
-		policies                  []*v1alpha1.VerificationPolicy
-		expected                  runtime.Object
-		expectedRefSource         *v1beta1.RefSource
+		name                       string
+		requester                  *test.Requester
+		verificationNoMatchPolicy  string
+		pipelinerun                v1beta1.PipelineRun
+		policies                   []*v1alpha1.VerificationPolicy
+		expected                   runtime.Object
+		expectedRefSource          *v1beta1.RefSource
+		expectedVerificationResult *trustedresources.VerificationResult
 	}{{
```

Comment:
https://google.github.io/styleguide/go/decisions#got-before-want
```suggestion
		wantPipeline                   runtime.Object
		wantRefSource          *v1beta1.RefSource
		wantExpectedVerificationResult *trustedresources.VerificationResult
	}{{
```

Reply:
I will update this in another PR for all trusted resources tests